docs/Model_Assumptions.rst

```diff
@@ -176,12 +176,85 @@ The confersion factors are derived from their `Gasoline Gallon Equivalents.<http
 Limitations
 -----------
 
+Currently, the MVS tool is still a work in progress. Running simulations with different component combinations might reveal some of its limitations. Some of the latter can be inherited while others can still be addressed. A limitation can result in drawbacks, but can also be beneficial in other perspectives. The following table lists the MVS limitations based on their type.
+
+.. list-table:: Limitations
```

Comment:
I think I liked the table as it was in your thesis better, with the two columns (inherited, to be adressed).

Reply:
I wanted to do it that way, but you also want a link. Unless I add the link next to the title of each limitation. Also in the presentation I listed 4 of each but in my report I had more. I am not sure if you want all of them but for now I did list all of them. If we keep all of them, the table will look unbalanced.

Reply:
An unbalanced table is ok, I think. What is printed when you insert `:ref`? I always get a compiler error, and online the same happens:

![grafik](https://user-images.githubusercontent.com/44204527/95068547-b81be980-0705-11eb-8a6b-3608e3cad3d6.png)

Reply:
@smartie2076 okay I will fix the table accordingly. For the references, they are also not working properly for me. I asked @mahendrark if I should keep -label or not. but both ways, the references are not printing..

Reply:
Do you maybe need a space between the `:ref` and the label itself?

Reply:
also doesn't work